doc/api/cli.md

```diff
@@ -628,6 +644,9 @@ file will be created if it does not exist, and will be appended to if it does.
 If an error occurs while attempting to write the warning to the file, the
 warning will be written to stderr instead.
 
+This can be overwritten with the use of the
```

Comment:
```suggestion
The `file` name may be an absolute path. If it is not, the default directory it will be written to is controlled by the
```

For consistency, there should be a `--warnings-dir` command line option (but that doesn't have to be added in this PR).